Package.swift

```diff
@@ -14,6 +14,7 @@ let package = Package(
     .package(url: "https://github.com/pointfreeco/swift-prelude.git", .revision("9a635ce")),
     .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .revision("c510e7d")),
     .package(url: "https://github.com/pointfreeco/swift-web.git", .revision("35421ef")),
+    .package(url: "https://github.com/pointfreeco/Ccmark.git", .branch("master")),
```

Comment:
we now have this repo https://github.com/pointfreeco/Ccmark for the cmark module map